### PR TITLE
Age

### PR DESCRIPTION
**Pull request by agebrouwer**
jo

### CODE REVIEW

File: api/src/routes/roles.js [hunk 1]
```diff
@@ -25,11 +25,7 @@ const connection = require('../config/db');
  */
 router.get('/getRole', validateSessionKey, (req, res) => {
     const sessionKey = req.headers.auth;
-    const sessionQuery = `SELECT rt.role_type
```
Comment (Jelle6837) [inline api/src/routes/roles.js:28]: bovenste behouden

File: app/lib/screens/login/login_screen.dart [hunk 2]
```diff
@@ -70,36 +60,33 @@ class _LoginPageState extends State<LoginPage> {
...
+    if (!ValidateUser.validateToken(code)) {
       setupErrorMessage = 'Code must be 6 characters!';
       setState(() {});
       return;
-
     }
-    
+
     try {
-      final exists = await APIService.checkCode(hashedCode);
```
Comment (Jelle6837) [inline app/lib/screens/login/login_screen.dart:83]: keep hashed code

File: app/lib/screens/login/login_screen.dart [hunk 3]
```diff
@@ -70,36 +60,33 @@ class _LoginPageState extends State<LoginPage> {
...
+      final exists = await APIService.checkCode(code);
```
Comment (Jelle6837) [inline app/lib/screens/login/login_screen.dart:71]: compare to hashed code

**Review by agebrouwer — COMMENTED** [review]
fixed some stuff

**Pull request closed without merging**